10up/10up-toolkit
# CSS Only Entrypoints

Related Issue/RFC: #168 

### Description of the Change






### Alternate Designs



### Possible Drawbacks



### Verification Process
1. `npm run dev`
2.  Add the Example block (WITHOUT LINARIA)
3. Change both style.css and editor.css.
4. Changes should hot-reload


### Checklist:



- [ ] I have read the [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document.
- [ ] I have added a changeset to my PR. See [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document for instructions
